src/app/assessment/assessment.component.ts

```diff
@@ -250,6 +250,21 @@ export class AssessmentComponent extends RouterEnter {
               ]
             });
           }
+          if (this.action === 'review' && !this.doReview) {
```

Comment:
This is not the correct condition for this situation, if action === 'review' and !this.doReview that means the user is seeing the review.

Reply:
@shawnm0705 should I just add the role to the condition?

Reply:
It’s nothing about checking the mentor role or not

The scenario is that the mentor is not assigned to the review, but he goes to the review page

The correct logic is
- API should not return the review data
- Front-end should pop up toast message if there’s no review data